modules/yaml/src/test/scala/pureconfig/module/yaml/YamlApiSuite.scala

```diff
@@ -4,12 +4,13 @@ import java.net.URLDecoder
 import java.nio.file.{ Files, Path, Paths }
 
 import com.typesafe.config.ConfigValue
+import org.scalatest.EitherValues
```

Comment:
Ditto on this one.

Reply:
`EitherValues` is something I added to work with `right.value` instead of `right.get` to get more meaningful error messages. Would you want me to refrain from using it?

Reply:
If you're using it, sounds good!